errors.js

```diff
@@ -0,0 +1,3 @@
+exports.serverErrorHandler = (err, req, res, next) => {
+  res.status(500).send({ msg: "Internal server error" });
+};
```

Comment:
I like that you've separated your errors, you could also put your endpoint "*" error handler in here too.